dynawo/sources/Modeler/DataInterface/DYNGeneratorInterfaceIIDM.cpp

```diff
@@ -95,6 +95,8 @@ GeneratorInterfaceIIDM::importStaticParameters() {
   staticParameters_.insert(std::make_pair("pMax", StaticParameter("pMax", StaticParameter::DOUBLE).setValue(pMax)));
   staticParameters_.insert(std::make_pair("qMax", StaticParameter("qMax", StaticParameter::DOUBLE).setValue(qMax)));
   staticParameters_.insert(std::make_pair("qMin", StaticParameter("qMin", StaticParameter::DOUBLE).setValue(qMin)));
+  staticParameters_.insert(std::make_pair("targetP_pu", StaticParameter("targetP_pu", StaticParameter::DOUBLE).setValue(getTargetP() / SNREF)));
+  staticParameters_.insert(std::make_pair("targetP_pu", StaticParameter("targetP", StaticParameter::DOUBLE).setValue(getTargetP())));
```

Comment:
I think it should be targetP instead of targetP_pu no ?

Reply:
thanks, I did a wrong report from a commit